www_src/blocks/image.jsx

```diff
@@ -1,20 +1,45 @@
-var React = require('react');
-var utils = require('../lib/propUtils');
+var React = require('react/addons');
 var assign = require('react/lib/Object.assign');
+var Spec = require('../lib/spec');
 
-var Image = React.createClass({
-  statics: {
-    defaults: {
-      src: '',
-      alt: '',
-      opacity: 1,
-      borderWidth: 0,
-      borderColor: 'transparent'
-    }
+var spec = new Spec('image', assign({
```

Comment:
This is just a set of allowed properties, including defaults, which allows for transformation between the API format and the react format -- I was super worried about doing transformations without having validations and tests on this